pkg/apis/crd/v1alpha1/types.go

```diff
@@ -354,10 +354,21 @@ type Rule struct {
 	// +optional
 	From []NetworkPolicyPeer `json:"from"`
 	// Rule is matched if traffic is intended for workloads selected by
-	// this field. If this field is empty or missing, this rule matches all
+	// this field. This field can't be used with ToService. If this field
+	// and ToService are both empty or missing, this rule matches all
 	// destinations.
 	// +optional
 	To []NetworkPolicyPeer `json:"to"`
+	// Rule is matched if traffic is intended for workloads selected by
+	// this field. Currently only ClusterIP and NodePort types Services
+	// are supported in this field. Workloads selected by this field are
+	// backend Endpoints+targetPorts of Services referred in this field.
+	// If the ServiceType is NodePort, workloads selected by this field
+	// also include the all Nodes+nodePort. This field can't be used with
+	// To or Ports. If this field and To are both empty or missing, this
```

Comment:
"If this field and To are both empty or missing" and its an ingress rule

Reply:
Do you mean egress rule?